import_precheck/README.md

```diff
@@ -0,0 +1,20 @@
+# Image Import Precheck Tool
+Precheck runs on your virtual machine before attempting to import it into
+Google Compute Engine (GCE). It attempts to identify compatibility issues that
+will either cause import to fail or will cause potentially unexpected behavior
+post-import. See our [image import documentation](https://googlecloudplatform.github.io/compute-image-tools/image-import.md)
+for more information.
+
+Precheck must be run as root/admin.
+
+## Binaries
+Windows: https://storage.googleapis.com/compute-image-tools/release/windows/import_precheck_d03684b55aa87913fc7608694279a189379322ab.exe
```

Comment:
This should be setup the same way we do Daisy so there is a release and head build and we always point to the release build here.

Reply:
Done.